pkg/networkmanager/network-interface-members.jsx

```diff
@@ -167,7 +158,7 @@ export const NetworkInterfaceMembers = ({
                             () => {
                                 return set_member(model, main_connection, main_connection.Settings,
                                                   cs.type, iface.Name, true)
-                                        .fail(show_unexpected_error);
+                                        .catch(show_unexpected_error);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19120-20230721-193803-d973134a-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)